lib/Pipeline.js

```diff
@@ -83,6 +83,7 @@ Pipeline.processJSON = function(gltf, options) {
  * @param {Object} [options.compressTextureCoordinates=false] Flag to run compressTextureCoordinates stage.
  * @param {Object} [options.kmcOptions=undefined] Options to pass to the generateModelMaterialsCommon stage, if undefined, stage is not run.
  * @param {Object} [options.quantize] Flag to run quantizeAttributes stage.
+ * @param {Boolean} [options.mergeVertices = false} Flag to merge duplicate vertices, which can produce a smaller model size but greatly increases conversion time.
```

Comment:
Maybe add this requires `options.preserve`. Same change in `lib/parseArguments`.

Or maybe we set `var shouldPreserve = defaultValue(options.preserve, (defaultValue(options.mergeVertices, false));`, but this could cause various other optimizations.

Or a 3rd option is to change
```
    if (!shouldPreserve) {
          if (options.mergeVertices) {
              mergeDuplicateVertices(gltfWithExtras);
          }
          MergeDuplicateProperties.mergeAll(gltfWithExtras);
          //....................
    }
```
to
```
    if (!shouldPreserve || options.mergeVertices) {
        mergeDuplicateVertices(gltfWithExtras);
    }
    if (!shouldPreserve) {
        MergeDuplicateProperties.mergeAll(gltfWithExtras);
        //....................
    }
```
PS. I also noticed that `options.preserve` is not documented here (it is documented in `lib/parseArguments').